run_dir/static/js/running_notes.js

```diff
@@ -123,7 +122,11 @@ function load_running_notes(wait) {
 function preview_running_notes(){
     var now = new Date();
     $('.todays_date').text(now.toDateString() + ', ' + now.toLocaleTimeString());
-    var category = generate_category_label($('.rn-categ button.active').text());
+    var categories = []
+    $('.rn-categ button.active').each(function() {
+      categories.push($(this).text());
```

Comment:
```suggestion
      categories.push($(this).text().trim());
```
Like in Line 231 below